vulcan/formula.py

```diff
@@ -75,14 +77,19 @@ def install(self, force=False):
                 shutil.rmtree(destination)
 
             # Move formula into place.
+            log.info('Moving formula %s to %s' %(self.name, destination))
             origin = os.path.join(tmpdir, self.origin_name)
             shutil.move(origin, destination)
 
             # Save state file.
             with open(self.state_file, 'w') as fh:
                 json.dump(self.as_dict(), fh)
 
+        except Exception as e:
```

Comment:
That makes sense, I was thinking maybe it was excepting and finally was masking it (wasn't 100% sure) - so its just an artifact of that testing.